app/models/language.rb

```diff
@@ -127,12 +125,11 @@ def self.score_lines(text)
   def self.update_recent_translations
     cutoff = @@last_update
     @@last_update = Time.zone.now
```

Comment:
```suggestion
  # We need a class variable here
  @@last_update = 1.minute.ago # rubocop:disable Style/ClassVars

  # Update I18n backend with any recent changes in translations.
  def self.update_recent_translations
    # We need a class variable here
    # rubocop:disable Style/ClassVars
    cutoff = @@last_update
    @@last_update = Time.zone.now
    # rubocop:enable Style/ClassVars
```
Gets rid of some suppressed RuboCop offenses